admin_manual/installation/system_requirements.rst

```diff
@@ -21,6 +21,14 @@ Web server        Apache 2.4 with ``prefork`` :ref:`apache-mpm-label` and ``mod_
 PHP Runtime       PHP (5.6+, 7.0, & 7.1)
 ================= =============================================================
 
+
+.. info:: 
+
+    For the future release of ownCloud 10.1, a minimum php version of 7.1 is needed.
+    If you use Ubuntu 16.04:
+    - php 7.1 is only available via ppa
```

Comment:
Can you link to a tutorial that shows how to use the ppa repos?

Reply:
sure. will do

Reply:
https://www.rosehosting.com/blog/install-php-7-1-with-nginx-on-an-ubuntu-16-04-vps/
https://askubuntu.com/questions/856793/upgrade-to-the-specific-php-7-1-7-2-from-php-7-0-in-ubuntu-16-04
You can reuse the php.ini files from php7.0 for php7.1 making the upgrade much easier. (own experience...)